src/Fieldtypes/Entries.php

```diff
@@ -165,9 +165,9 @@ protected function getFirstCollectionFromRequest($request)
 
     public function getSortColumn($request)
     {
-        $column = $request->get('sort');
+        $column = $request->get('sort', 'title');
 
-        if (! $column && ! $request->search) {
+        if (! $request->sort && ! $request->search && count($this->config('collections')) < 2) {
```

Comment:
Instead of `$this->config('collections')` use `$this->getConfiguredCollections()`